strings/jaro_winkler.py

```diff
@@ -34,25 +34,30 @@ def get_matched_characters(_str1: str, _str2: str) -> str:
                 matched.append(l)
                 _str2 = f"{_str2[0:_str2.index(l)]} {_str2[_str2.index(l) + 1:]}"
 
-        return ''.join(matched)
+        return "".join(matched)
 
     # matching characters
     matching_1 = get_matched_characters(str1, str2)
     matching_2 = get_matched_characters(str2, str1)
     match_count = len(matching_1)
 
     # transposition
-    transpositions = len(
-        [(c1, c2) for c1, c2 in zip(matching_1, matching_2) if c1 != c2]
-    ) // 2
+    transpositions = (
+        len([(c1, c2) for c1, c2 in zip(matching_1, matching_2) if c1 != c2]) // 2
```

Comment:
This line has a length of 82 chars, is it below our threshold?

Reply:
Black __max_line_length__ is 88